packages/shared/src/components/squads/SquadFeedHeading.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { ReactElement, useContext, useMemo } from 'react';
+import { Button } from '../buttons/Button';
+import { PinIcon } from '../icons';
+import { Squad } from '../../graphql/sources';
+import { ActiveFeedContext } from '../../contexts';
+import { useSquadActions } from '../../hooks';
+import { LoggedUser } from '../../lib/user';
+
+interface SquadFeedHeadingProps {
+  squad: Squad;
+  user: LoggedUser;
+}
+
+function SquadFeedHeading({
+  squad,
+  user,
+}: SquadFeedHeadingProps): ReactElement {
+  const { items } = useContext(ActiveFeedContext);
+  const { collapseSquadPinnedPosts, expandSquadPinnedPosts } = useSquadActions({
+    squad,
+    user,
+  });
+
+  const onClick = async () => {
+    return squad.currentMember.flags.collapsePinnedPosts
+      ? await expandSquadPinnedPosts(squad.id)
+      : await collapseSquadPinnedPosts(squad.id);
+  };
+
+  const pinnedPostsCount = useMemo(() => {
+    let countPinned = 0;
+    for (const item of items) {
+      if (item.type === 'post' && !!item.post.pinnedAt) {
+        countPinned += 1;
+      } else {
+        break;
+      }
+    }
+
+    return countPinned;
+  }, [items]);
+
+  return (
+    <div className="flex flex-row flex-wrap gap-4 justify-end items-center pb-2 w-full">
+      <span className="flex flex-row gap-3 pl-3 ml-auto border-l border-theme-divider-tertiary">
+        <Button
+          className="btn-tertiaryFloat"
```

Comment:
The button does not match the styling exactly, the alternative is `btn-secondary` but that does not match the design spec either:

<img width="426" alt="Screenshot 2023-10-23 at 17 36 18" src="https://github.com/dailydotdev/apps/assets/1225100/5328c80d-0746-402d-8069-12243a974334">
<img width="426" alt="Screenshot 2023-10-23 at 17 36 27" src="https://github.com/dailydotdev/apps/assets/1225100/219b665b-a423-4c0d-987e-d1b9b9527506">

Reply:
Non existing button I think, I'm [asking design](https://dailydotdev.slack.com/archives/C05LMGUD8HZ/p1698127221954679) if we can use the tertiaryFloat for it.

Reply:
So [end](https://dailydotdev.slack.com/archives/C05LMGUD8HZ/p1698129789642549?thread_ts=1698127221.954679&cid=C05LMGUD8HZ) decision is to use `btn-tertiaryFloat` as you proposed.